RIS.js

```diff
@@ -6393,6 +6420,55 @@ var testCases = [
 				"seeAlso": []
 			}
 		]
+	},
+	{
+		"type": "import",
+		"input": "TY  - JOUR\nT1  - Reputation and Decision Making under Ambiguity: A Study of U.S. Venture Capital Firms' Investments in the Emerging Clean Energy Sector\nAU  - Petkova, Antoaneta P.\nAU  - Wadhwa, Anu\nAU  - Yao, Xin\nAU  - Jain, Sanjay\nY1  - 2013/05/28\nPY  - 2013\nDA  - 2014/04/01\nN1  - doi: 10.5465/amj.2011.0651\nDO  - 10.5465/amj.2011.0651\nT2  - Academy of Management Journal\nJF  - Academy of Management Journal\nJO  - AMJ\nSP  - 422\nEP  - 448\nVL  - 57\nIS  - 2\nPB  - Academy of Management\nSN  - 0001-4273\nM3  - doi: 10.5465/amj.2011.0651\nUR  - https://doi.org/10.5465/amj.2011.0651\nY2  - 2020/02/19\nER  - ",
```

Comment:
To clarify the behavior under test here, maybe use a test case with just a regular tag and a deprecated tag? (I'm not even clear on what's being tested in the input. The `Y2`? That doesn't seem to have been added to Extra.)

Reply:
Oh, true. It looks like it actually goes in `item.unsupportedFields`, which ends up nowhere, unless the true-by-default `RIS.import.ignoreUnknown` preference is set to false (in which case it goes into a note). I could add special handling for deprecated tags, but I'm unsure about whether the information would be meaningful - the RIS spec is clear that deprecated tags are always subordinate to their non-deprecated equivalents.

Reply:
I'm OK with discarding those, then. But let's still clarify the test.

Reply:
OK, done.